src/wallet/wallet2.cpp

```diff
@@ -11657,6 +11678,34 @@ void wallet2::check_tx_key_helper(const crypto::hash &txid, const crypto::key_de
   }
 }
 
+bool wallet2::is_out_to_acc(const cryptonote::account_public_address &address, const crypto::public_key& out_key, const crypto::key_derivation &derivation, const std::vector<crypto::key_derivation> &additional_derivations, const size_t output_index, crypto::key_derivation &found_derivation, const boost::optional<crypto::view_tag> view_tag_opt) const
```

Comment:
Just an idea: You have some nice comments in the called `out_can_be_to_acc` method itself already about the general working principle of the view tags, but because this here is the crucial `wallet2` method where the core of the new wonderful magic sits, maybe you could add some comments here as well.